Packs/McAfeeNSM/Integrations/McAfeeNSMv2/McAfeeNSMv2.py

```diff
@@ -0,0 +1,1975 @@
+                            "ApplicationType": None
+                        }
+                    ],
+                    "ApplicationObjectList": [],
+                    "TimeObjectList": [
+                        {
+                            "RuleObjectId": "-1",
+                            "Name": "Always",
+                            "RuleObjectType": None
+                        }
+                    ]
+                }
+            ]
+        }
+    }
+
+
+def create_body_create_rule(rule_type: str, address: List, number: int,
+                            from_to_list: list[dict[str, Any | None]]) -> tuple:
+    """ create part of the body for the command create_rule_object
+        Args:
+            rule_type: str - The type of the rule.
+            address: List - A list of addresses, if relevant.
+            number: int - The number of the IPV.
+            from_to_list: List - A list that contains dictionaries with from and do addresses.
+        Returns:
+            Returns the body for the request.
+        """
+    if 'HOST' in rule_type:
+        return f'HostIPv{number}', {
+            f'hostIPv{number}AddressList': address
+        }
+    elif 'ADDRESS_RANGE' in rule_type:
+        return f'IPv{number}AddressRange', {
+            f'IPV{number}RangeList': from_to_list
+        }
+    else:
+        return f'Network_IPV_{number}', {
+            f'networkIPV{number}List': address
+        }
+
+
+def check_args_create_rule(rule_type: str, address: List, from_address: str, to_address: str, number: int):
+    """ Validate the arguments of the function
+        Args:
+            rule_type: str - The type of the rule.
+            address: List - A list of addresses, if relevant.
+            from_address: str - The from address, if relevant.
+            to_address: str - The to address, if relevant.
+            number: int - The number of the addresses IP V.
+    """
+    if ('4' in rule_type and number == 6) or ('6' in rule_type and number == 4):
+        raise Exception('The version of the IP in "rule_object_type" should match the addresses version.')
+    if ('HOST' in rule_type or 'NETWORK' in rule_type) and (not address or from_address or to_address):
+        raise Exception(f'If the "rule_object_type" is “Endpoint IP V.{number}” or “Network IP V.{number}” than only'
+                        f' the argument “address_ip_v.{number}” must contain a value. The other address arguments '
+                        f'should be empty.')
+    if 'ADDRESS_RANGE' in rule_type and (not to_address or not from_address or address):
+        raise Exception(f'If the "rule_object_type" is “Range IP V.{number}” than only the arguments '
+                        f'“from_address_ip_v.{number}” and “to_address_ip_v.{number}” must contain a value, the other'
+                        f' address arguments should be empty.')
+
+
+def add_entries_to_alert_list(alert_list: List[Dict]) -> List[Dict]:
```

Comment:
Why do we need those commands? why not run directly on the response and create the dict there? 
This is for all the add_entry/update_entries functions

Reply:
I thought that it will make the actual command function shorter and easier to read. I can remove the extra function and do it directly on the response if it is better